### PR TITLE
Merge release branch

### PR DESCRIPTION
**Pull request by ashb**
It turns out we had deployed this "release" branch on the live server, but nothing referenced this branch. Lets just merge it back to master.

### CODE REVIEW

**Comment by ashb** [comment]
@alexbowen @frank-long One of you are sort of best placed to review this one.

It's no changes of my own, just Kevin's previous changes merging into master branch -- I think we want to use master for deployments?

File: package.json [hunk 1]
```diff
@@ -58,8 +59,6 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "postinstall": "bower install",
-    "test": "grunt test",
-    "start": "grunt serve:dist"
+    "test": "grunt test"
```
Comment (dhatanian) [inline package.json]: @ashb what's the script used to start the app, if there is only `test` in `package.json`

Comment (ashb) [inline package.json]: This was just Kevin's changes from last year, so 🤷‍♂️. I also think that in deployment we run out of the `dist/` folder, and having `npm` or `grunt` process running isn't needed or encouraged.

Comment (ashb) [inline package.json]: Do you think it's worth adding it back anyway?

Comment (dhatanian) [inline package.json]: No that's fine, but I don't have the rights to merge this PR. I think @samhilluk is planning to fix this issue.

Comment (alexbowen) [inline package.json]: ill merge if there is another/preferred way to start the app

**Pull request merged**